guides/micronaut-cache-hazelcast/src/main/resources/application.yml

```diff
@@ -0,0 +1,18 @@
+micronaut:
+  application:
+    name: micronautguide
+---
+#tag::config[]
+micronaut:
+  caches:
+    headlines: # <1>
+      charset: 'UTF-8'
+#end::config[]
+---
+#tag::hazelcast[]
+hazelcast:
+  client:
+    network:
+      addresses:
+        - '192.168.4.67:5701'
```

Comment:
I start my docker container with:

```
docker run -it --rm -p 5701:5701 hazelcast/hazelcast:4.2.1
```

And then I put here `127.0.0.1:5701`.

Reply:
For me, it does not work.